### PR TITLE
Space travelers Hub

### PR DESCRIPTION
**Pull request by mikemtzp**
Developed a Space travelers hub app where you can reserve both rockets and missions and see your reserved travels.

- Set up with React app and Redux
- Create routes and view components (Rockets, Missions, My Profile). 
- Create directories for all Redux state slice files (rockets and missions).
- Create Rocket and Mission components.
- Add styling using SCSS.
- Fetch data from the [Rockets endpoint](https://api.spacexdata.com/v3/rockets) and [Missions endpoint](https://api.spacexdata.com/v3/missions).
- Generate reducer for both Missions and Rockets and connect it with React through the store.
- Use useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes.
- Create an action that is dispatched when a user wants to reserve a rocket or join a mission updating the state in the respective store.
- Add conditional rendering to show a different button when the state is updated.
- Add tests for components and state.

## [See Live Deployment](https://app.netlify.com/sites/spacetravelers-hub/settings/domain)

### CODE REVIEW

File: src/pages/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          <h2 className="mycontainer-section__title">
+            My Rockets
+          </h2>
+          <ul className="mycontainer-section__box">
+            {
+              reservedRockets.length ? reservedRockets.map((item) => (
+                <li key={item.id} className="element-name">{item.name}</li>
+              )) : <li className="element-name">No rockets selected</li>
+            }
+          </ul>
```
Comment (zieeco) [inline src/pages/MyProfile.js:57]: - Hey, great job with the project, you've done well so far, however, there's a part of the project requirement that has not been implemented yet. please kindly ensure you've use the `filter` method to render your rocket and mission to meet the project requirement below::

filter

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mikemtzp,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Nice code organization. ✔️
- Good git-flow usage ✔️
- Good commits message. ✔️





## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/MyProfile.js (hunk 1 above)
Comment (mikemtzp) [reply src/pages/MyProfile.js]: Hi @zieeco! 😁 
Thank you very much for the review! The refactorization using the `filter()` to render rockets and missions in the `My Profile` page is now working.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @mikemtzp @Zeraltz  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**